### ISSUE TITLE
Changes in May 2019 Windows update cause crackle and distortion

### ISSUE THREAD

**Issue by paulheu** [eiz/SynchronousAudioRouter#70]
Something changed in the current windows update which causes severe crackling and pops in SAR. I notice that heavy network activity will also causes these and sometimes throws audio out of sync.

Never had these issues prior and as there have been a number of changes to the audio subsystem in this update that obviously plays into it.

Would be really nice to hear anything on his, have others see this and/or is there a fix coming?

**Comment by amurzeau** [comment]
If network activity cause audio issues, I guess the network driver lock the cpu for too much time under load and increase system latency.
Does the network driver get updated at the same time ? What's the update version and your previous one ?

**Comment by cjunekim** [comment]
Yes. This is a well known issue. Windows update 1903 causes ntoskrnl.exe to take a peak DPC latency over 1ms. Many user attempts turned out to be failing. Currenly, the only solution is reverting back to 1809. I guess MS is working on this.
